static/style.css

```diff
@@ -1,5 +1,5 @@
 
-@import url('./vendor/uswds-1.0.0/css/uswds.min.css');
+@import url('./vendor/uswds-1.0.0/css/uswds.min.css'); 
```

Comment:
I think this change should be discarded since it is just an unnecessary space at the end of a line